repo-issues/lib/data-source.js

```diff
@@ -0,0 +1,97 @@
+const http = require('request-promise-native')
+const moment = require('moment')
+const octokit = require('@octokit/rest')()
+
+const show_details_threshold = 5
+const organisation = 'novoda'
+const languageIconMap = {
+    "html": "html5",
+    "groovy": "gradle",
+    "css": "css3",
+    "objective-c" : "c"
+}
+
+let showIssues = false
+let showPRs = true
+
+const generateViewState = (configuration) => {
+    showIssues = configuration.showIssues
+    showPRs = configuration.showPRs
+
+    octokit.authenticate({type: 'oauth', token: `${configuration.token}`})
+    
+    return octokit.repos.getForOrg({ org: organisation, type: 'all', per_page: 9999 })
+        .then(repos => Promise.all(repos.data.map(queryRepoIssues)))
+        .then(prs => prs.filter(each => each.prs.length > 0 || each.issues.length > 0))
+        .then(toViewState)
+}
+
+function queryRepoIssues(repo) {
+    return octokit.issues.getForRepo({ owner: organisation, repo: repo.name })
+        .then(result => toRepoIssues(result, repo))
+}
+
+function toRepoIssues(result, repo) {
+    const allitems = { prs: [], issues: [] }
+
+    result.data
+        .filter(each =>
+            (showPRs && each.pull_request !== undefined) ||
+            (showIssues && each.pull_request == undefined))
```

Comment:
⚠️ Shouldn't this be a triple equals given the previous is a `!==` (negated identity)? Or, the other one should be `!=` instead, but I think this is the most likely to be wrong (for my limited JS knowledge).